### ISSUE TITLE
Confusing strings

### ISSUE THREAD

**Issue by jermanuts** [m2049r/xmrwallet#968]
Both strings`send_send_label` and `send_send_timed_label` are confusing

https://github.com/m2049r/xmrwallet/blob/989d52b33de200253f1ea3bc2d7d91b8764079fd/app/src/main/res/values/strings.xml#L221-L222

"Spend my sweet Moneroj"

It sounds as if "Moneroj" is the actual cryptocurrency coin and not Monero.

**Comment by anhdres** [comment]
You're correct. It should be either "moneroj" (Esperanto) or "moneros" (English) with lower case if we're talking about the units or "Monero" with upper case (the currency).

Maybe in languages like German when you capitalize nouns makes sense to use Moneroj, otherwise makes no sense IMO.

**Comment by jermanuts** [comment]
Should be "Monero", this is a very bad UX to be called moneroj IMO.

**Comment by anhdres** [comment]
You're probably right. I'm a bit afraid that if we strip Monerujo of all its little quirks it'd be left a bit generic, but we'll discuss it.
